bridgeclient/config.js

```diff
@@ -0,0 +1,5 @@
+// Bridge applications
+module.exports = {
+    APP_ETH_CONTRACT_ADDRESS: '0x0823eFE0D0c6bd134a48cBd562fE4460aBE6e92c',
```

Comment:
In the E2E test we'll likely shell out to a truffle script to retrieve the address. So I wouldn't include these hardcoded addresses.

Instead, can we update the `BridgeClient` constructor so that it accepts addresses? Something like:

```js
new BridgeClient(endpoint, ethAppAddr, erc20AppAddr)
```

Reply:
Added in https://github.com/Snowfork/polkadot-ethereum/pull/108/commits/66eee87936fb9bdf7518112254a0fe7549e8c7b2